x/nft/CHANGELOG.md

```diff
@@ -28,3 +28,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 -->
 
 # Changelog
+
+## [Unreleased]
+
+### Features
+
+* [#17825](https://github.com/cosmos/cosmos-sdk/pull/17825) Add AutoCLI Options.
```

Comment:
```suggestion
*(x/nft) [#17825](https://github.com/cosmos/cosmos-sdk/pull/17825) Add AutoCLI Options.
```

Reply:
This isn't necessary as it's in the nft changelog